README.md

```diff
@@ -23,51 +23,32 @@ also be useful in other applications. For example, proctoring applications could
 use it to detect and /request that users disable background blur to ensure
 proper monitoring.
 
-The proposed solution exposes the read-only effect state on MediaStreamTrack and
-VideoFrameMetadata. This will enable Web developers to easily determine the
-current state of the effect and provide event handlers for change detection.
+The proposed solution exposes the read-only effect state on VideoFrameMetadata.
+This will enable Web developers to easily determine the current state of the
+effect.
 
 ## Goals
 
-* Allow Web developers to easily access and monitor changes in platform blur.
-* Enable Web developers to build new features that respond to changes in
-    background blur.
-* Provide a consistent and easy-to-use API for accessing platform effect state.
+- Allow Web developers to easily access and monitor changes in platform blur.
+- Enable Web developers to build new features that respond to changes in
+  background blur.
+- Provide a consistent and easy-to-use API for accessing platform effect state.
 
 ## Non-goals
 
-* This API does not provide a way to control platform effects. That
-    functionality may be exposed in a future API.
-* This API does not attempt to polyfill effects in platforms/browsers that do
-    not support them.
-* This API doesn’t include all possible platform effects. More effects may be
-   exposed as future extensions of the API.
-
-## MediaStreamTrack
-
-The effect has an associated `MediaEffect` property on the `MediaStreamTrack`
-interface. The state of the effect can be obtained from the property. Any change
-to the state fires an event on the effect object. The presence of the field can
-also be used to detect if the platform/browser supports background blur.
-
-```js
-const stream = await navigator.mediaDevices.getUserMedia({ video: true });
-const videoTrack = stream.getVideoTracks()[0];
-if (videoTrack.backgroundBlur) {
-  const effect = videoTrack.backgroundBlur;
-  console.log("Background blur state:", effect.state);
-  effect.addEventListener("change", (event) => {
-    console.log("Background blur state changed:", event.target.state);
-  });
-}
-```
+- This API does not provide a way to control platform effects. That
+  functionality may be exposed in a future API.
```

Comment:
The control piece is already available through `applyConstraints()`, although there is room for improvement there.
```suggestion
- This API does not provide a way to control platform effects.
```